Utilities/CodeGeneration/EdFi.Ods.CodeGen/Generators/DatabaseViews.cs

```diff
@@ -11,9 +11,9 @@ namespace EdFi.Ods.CodeGen.Generators
 {
     public class DatabaseViews : GeneratorBase
     {
-        private readonly IViewsProvider _viewsProvider;
+        private readonly IAuthorizationDatabaseTableViewsProvider _viewsProvider;
```

Comment:
Rename of the interface is great, but update the field and parameter on the constructor to also take into account the rename.

Reply:
I fixed it .Thanks